next.config.js

```diff
@@ -3,6 +3,13 @@ const withImages = require("next-images");
 
 module.exports = withImages({
 	reactStrictMode: true,
+	webpack(config) {
+		// replace react with preact to reduce bundle size and faster DevServer
+		Object.assign(config.resolve.alias, {
+			"react-dom": "preact/compat",
+			react: "preact/compat",
+		});

```

Comment:
you have to return a webpack config in this function. If you fix that i can merge :)

Reply:
sorry! I added.